Tests/UnitTests/APIToRequestMappingTests.swift

```diff
@@ -170,6 +170,20 @@ class APIToRequestMappingTests: XCTestCase {
         XCTAssert(try element.enabled == true)
     }
 
+    func testSessionScript() throws {
+        let mockWebDriver = MockWebDriver()
+        let session = Session(webDriver: mockWebDriver, existingId: "mySession")
+        mockWebDriver.expect(path: "session/mySession/execute", method: .post)
```

Comment:
Ideally also add a closure to assert that the request body contains the expected args.